pkg/cri/resource-manager/cache/cache.go

```diff
@@ -694,6 +696,35 @@ func (cch *cache) LookupContainer(id string) (Container, bool) {
 	return c, ok
 }
 
+// LookupContainerByCgroup looks up the container for the given cgroup path.
+func (cch *cache) LookupContainerByCgroup(path string) (Container, bool) {
+	cch.Debug("resolving %s to a container...", path)
+
+	for id, c := range cch.Containers {
+		if id != c.CacheID {
+			continue
+		}
```

Comment:
When do we hit this, again? Maybe a short comment to clarify(?)

Reply:
We hash containers by two IDs: the ID in the CRI runtime and our cache ID. We hit this for every container which has already been successfully created in the actual runtime. So it is rather the rule, not the exception.

Reply:
Ah yeah, of course